libut/Makefile

```diff
@@ -1,33 +1,41 @@
-OBJS=libut.a
-all: $(OBJS) 
+SHELL=/usr/sh
+VERSION=2.0.1
+MAJOR=2
```

Comment:
Nit: I'd prefer to see these variables inlined into the places where they're currently being used, to keep the code as straightforward as possible. I infer from context that `MAJOR` is always supposed to be the major version, e.g. if `VERSION` became `3.0.0` then `MAJOR` should become `3`, but I don't really understand why. This would probably be easier to understand if half of the logic weren't located 30 lines away.